examples/tree_2d_dgsem/elixir_euler_sedov_blast_wave_MCL.jl

```diff
@@ -94,7 +94,8 @@ callbacks = CallbackSet(summary_callback,
 ###############################################################################
 # run the simulation
 
-stage_callbacks = (BoundsCheckCallback(save_errors=false),)
+output_directory = "out"
+stage_callbacks = (BoundsCheckCallback(save_errors=true, interval=100, output_directory = output_directory),)
```

Comment:
I believe "out" should be the default output directory. Why is this needed in the elixir?
```suggestion
stage_callbacks = (BoundsCheckCallback(save_errors=true, interval=100),)
```

Reply:
I added it for the case that someone adjusts `output_directory` in the elixir file. Then, we would save our file to this location and the check fails since `deviation.txt` is not found.
Like I implemented it, for the test, `output_directory` from the elixir is always overwritten and set to "out".

Reply:
But I agree with you that it's a bit weird. If someone adjusts `save_errors=false` or just sets `interval` to `5` or smaller, the test fails anyway.

Reply:
ok!